bitmap-team-deuces/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
This is great!  Though I noticed one minor detail which could be improved, see my comment below.